### PR TITLE
[WIP]add brainfcuk layer

### PR DESCRIPTION
**Pull request by JAremko**
The Brainfuck language is notoriously hard to be productive with. The layer attempts to solve this problem and defeat the whole purpose of the language...

### CODE REVIEW

**Comment by d12frosted** [comment]
At last I can learn Brainfuck! Thanks, @JAremko :smile_cat:

**Comment by JAremko** [comment]
@d12frosted  Actually, I  [wasn't for realz](https://github.com/syl20bnr/spacemacs/pull/5652/files). But now it looks like I am gonna be adding it :smile:

File: layers/+lang/brainfuck/packages.el [hunk 1]
```diff
@@ -0,0 +1 @@
+(setq brainfuck-packages '(brainfuck-mode))(defun brainfuck/init-brainfuck-mode()(use-package brainfuck-mode :defer t :init(spacemacs/set-leader-keys-for-major-mode 'brainfuck-mode "bf" 'bf-help-describe-symbol)))
```
Comment (robbyoconnor) [inline layers/+lang/brainfuck/packages.el]: Is this shit intentional?

Comment (JAremko) [inline layers/+lang/brainfuck/packages.el]: @robbyoconnor   Most likely. 
It was still April 1, somewhere. :wink:

**Comment by JAremko** [comment]
I think I'll hide it under the rug for a while.  PR police is after me,  gotta lay low.

**Comment by robbyoconnor** [comment]
NOOO reopen it -- I wanna see the world burn. @syl20bnr  --please merge this

**Comment by JAremko** [comment]
@robbyoconnor  Too late! I delete the branch :smiling_imp:

**Comment by robbyoconnor** [comment]
:trollface:

File: layers/+lang/brainfuck/packages.el (hunk 1 above)
Comment (robbyoconnor) [inline layers/+lang/brainfuck/packages.el]: THIS NEEDS TO BE USED!

**Comment by robbyoconnor** [comment]
DAMN IT I WANT THIS BACK

**Pull request closed without merging**